src/chart_types/heatmap/state/selectors/tooltip.ts

```diff
@@ -48,9 +48,37 @@ export const getTooltipInfoSelector = createCachedSelector(
       pickedShapes
         .filter(({ visible }) => visible)
         .forEach((shape) => {
-          const xValueLabel = config.xAxisLabel.formatter(shape.datum.x);
+          // X-axis value
           tooltipInfo.values.push({
-            label: `${shape.datum.y} - ${xValueLabel}`,
+            label: config.xAxisLabel.name,
+            color: RGBtoString(shape.fill.color),
```

Comment:
```suggestion
            color: 'transparent',
```
I think X and Y values should not be connected to the cell color

Reply:
Fixed in https://github.com/elastic/elastic-charts/pull/847/commits/b8081acaaaad1740022c85d4912c52d0ea80085e